lib/dialects/sqlite3/index.js

```diff
@@ -199,7 +199,19 @@ class Client_SQLite3 extends Client {
         return [ctx.lastID];
       }
       case 'del':
-      case 'update':
+      case 'update': {
+        if (returning) {
+          if (response) {
+            return response;
+          }
+
+          // ToDo Implement after https://github.com/microsoft/vscode-node-sqlite3/issues/15 is resolved
+          this.logger.warn(
```

Comment:
is this still relevant? fairly sure it already works

Reply:
could you please do that?

Reply:
done